src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditor.js

```diff
@@ -281,9 +283,10 @@ class OrderTemplatesEditor extends Component {
                     label={ORDER_TEMPLATES_ACCORDION_TITLES[ORDER_TEMPLATES_ACCORDION.POL_FUND_DISTIBUTION]}
                     id={ORDER_TEMPLATES_ACCORDION.POL_FUND_DISTIBUTION}
                   >
-                    <POLineFundDistributionForm
```

Comment:
`POLineFundDistributionForm` contains some props, like `required={false}`. Are you sure that it will stay the same?

Reply:
You are right. fixed